### PR TITLE
Add default config values

### PR DESCRIPTION
**Pull request by ovolynets**

### CODE REVIEW

File: app/org/zalando/markscheider/MetricsPlugin.scala [hunk 1]
```diff
@@ -72,7 +72,7 @@ class MetricsPlugin @Inject() (
 
     if (enabled) {
       val rateUnit = configuration.getString("org.zalando.markscheider.rateUnit", validUnits).getOrElse("SECONDS")
-      val durationUnit = configuration.getString("org.zalando.markscheider.durationUnit", validUnits).getOrElse("MILLISECONDS")
+      val durationUnit = configuration.getString("org.zalando.markscheider.durationUnit", validUnits).getOrElse("SECONDS")
```
Comment (lenalebt) [inline app/org/zalando/markscheider/MetricsPlugin.scala]: That would possibly break clients (our, for example :)), please do not change it. You can override this value in your clients through configuration then.

File: conf/reference.conf [hunk 2]
```diff
@@ -1,4 +1,11 @@
 org.zalando.markscheider {
+
+  durationUnit = "SECONDS"
```
Comment (lenalebt) [inline conf/reference.conf]: here as well, please do not change it to seconds. Change in your service through configuration.

File: app/org/zalando/markscheider/MetricsPlugin.scala (hunk 1 above)
Comment (ovolynets) [inline app/org/zalando/markscheider/MetricsPlugin.scala]: I see. I wanted to make it consistent with the defaults from metrics-play but will change it back.

Comment (lenalebt) [inline app/org/zalando/markscheider/MetricsPlugin.scala]: At least our internal version had the same settings as far as I can see.

**Comment by lenalebt** [comment]
+1

**Pull request merged**